src/sql/src/statements/insert.rs

```diff
@@ -25,6 +25,10 @@ pub struct Insert {
 }
 
 impl Insert {
+    pub fn new(inner: Statement) -> Self {
```

Comment:
Do we still need this constructor methods in light that `inner` is already public?